plugin.py

```diff
@@ -136,7 +138,37 @@ def on_pre_start(
         configuration.command = cls.get_command()
         return None
 
-    # notification handlers
+    # -- commands from the server that should be handled client-side ----------
+
+    def on_pre_server_command(
+        self,
+        command: Mapping[str, Any],
+        done_callback: Callable[[], None]
+    ) -> bool:
+        name = command["command"]
+        if name == "omnisharp/client/findReferences":
+            return self._handle_quick_references(command["arguments"], done_callback)
+        return False
+
+    def _handle_quick_references(self, arguments: List[Any], done_callback: Callable[[], None]) -> bool:
+        session = self.weaksession()
+        if not session:
+            return False
+        sb = session.get_session_buffer_for_uri_async(arguments[0]["uri"])
+        if not sb:
+            return False
```

Comment:
Triggering `lsp_symbol_references` will only work when done on currently open file. Is it guaranteed that this command will only be triggered for the current file?

Reply:
I don't know. Let's wait until it's a problem for someone.